.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
     },
     "eslint.enable": false,
     "flow.enabled": false,
-
+    "editor.formatOnSave": true,
```

Comment:
@mennenia @sarahscott @alloy @l2succes - this turns on prettier by default when you press save